File: Java/#1304_sumZero.java
class Solution {
    public int[] sumZero(int n) {
        int[] res = new int[n];
        int i=0;
        int j=n-1;
        while(i<j){
            res[i]=i+1;
            res[j]=-res[i];
            i++;
            j--;
        }
        return res;
    }
}

File: Python/#1304_sumZero.py
class Solution:
    def sumZero(self, n: int) -> List[int]:
        res = [0]*n
        i = 0
        j = n-1
        while i<j:
            res[i] = n-i
            res[j] = -res[i]
            i += 1
            j -= 1
        return res
